README.md

```diff
@@ -6,24 +6,38 @@ A [Vyper](https://github.com/vyperlang/vyper) to Cairo transpiler, inspired by [
 
 As this is a work in progress, there are numerous Vyper types (e.g. static arrays, dynamic arrays) and features that are not supported yet. Some features are also not capable of being supported on StarkNet. In these cases, the transpiler will throw an error.
 
-## Installation
+## Getting started
 
-### Prequisites
+### Dependencies
 
-#### Python
+Vyro currently only supports Python 3.9 due to the Python version constraint of the libraries we rely on. Follow the instructions to install the latest version of python for your platform in the [python docs](https://docs.python.org/3/using/unix.html#getting-and-installing-the-latest-version-of-python)
 
-Vyro requires Python 3.9. Follow the instructions to install the latest version of python for your platform in the [python docs](https://docs.python.org/3/using/unix.html#getting-and-installing-the-latest-version-of-python)
+### Building from source
 
-#### Virtual Environment
+If you are building from source, either because you want to contribute or just that you want to explore the codebase, we reccomend you work within a virtual environment. Since we utilize [poetry](https://python-poetry.org/docs/#installation) as our dependency management system, you can easily setup your virtual environment by doing this at the root of the project.
```

Comment:
```suggestion
If you are building from source, either because you want to contribute or explore the codebase, we recommend you work within a virtual environment. Since we utilize [poetry](https://python-poetry.org/docs/#installation) as our dependency management system, you can easily setup your virtual environment by doing this at the root of the project:
```